.dotnet/src/Custom/FineTuning/FineTuningJob.cs

```diff
@@ -0,0 +1,23 @@
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace OpenAI.FineTuning;
+
+[CodeGenModel("FineTuningJob")]
+public partial class FineTuningJob
+{
+    internal static FineTuningJob FromResponse(PipelineResponse pipelineResponse)

```

Comment:
I think this `FromResponse` should generate in the same form. If it doesn't, let us know and we should fix it!

Reply:
Not sure what you mean here